tests/integration/datadictwithobjid/submission/test_upload.py

```diff
@@ -417,6 +417,77 @@ def get_index_by_uuid(uuid):
     assert entity['action'] == 'update'
 
 
+@patch('sheepdog.transactions.upload.sub_entities.FileUploadEntity.get_file_from_index_by_hash')
+@patch('sheepdog.transactions.upload.sub_entities.FileUploadEntity.get_file_from_index_by_uuid')
+@patch('sheepdog.transactions.upload.sub_entities.FileUploadEntity._create_index')
+@patch('sheepdog.transactions.upload.sub_entities.FileUploadEntity._create_alias')
+@patch('sheepdog.transactions.upload.sub_entities.FileUploadEntity._update_acl_uploader_for_file')
```

Comment:
I think here it might make sense to mock the entire `FileUploadEntity`?

Reply:
we still want to go through the normal update flow